scripts/benchmarks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+steps=50
+repeat=20
+canvasOutput=./runtime/src/weights
+canvasChain=canvas-dev
+pallets=(
+	pallet_balances
+	pallet_collator_selection
+	pallet_contracts
+	pallet_session
+	pallet_timestamp
+)
+
+for p in ${pallets[@]}
+do
+	./target/release/canvas benchmark \
```

Comment:
```suggestion
	cargo run --release --features=runtime-benchmarks -- benchmark \
```

This is probably okay with the caching between runs, but if not the script should at least build the binary beforehand instead of making the caller of the script do that themselves